pay-api/src/pay_api/services/eft_service.py

```diff
@@ -47,6 +52,17 @@ def create_account(self, identifier: str, contact_info: Dict[str, Any], payment_
         cfs_account.status = CfsAccountStatus.PENDING.value
         return cfs_account
 
+    def update_account(self, name: str, cfs_account: CfsAccountModel, payment_info: Dict[str, Any]) -> CfsAccountModel:
+        """Update pad account."""
+        if str(payment_info.get('bankInstitutionNumber')) != cfs_account.bank_number or \
+                str(payment_info.get('bankTransitNumber')) != cfs_account.bank_branch_number or \
+                str(payment_info.get('bankAccountNumber')) != cfs_account.bank_account_number:
+            # This means, the current cfs_account is for PAD, not EFT
+            # Make the current CFS Account as INACTIVE in DB
+            cfs_account.status = CfsAccountStatus.INACTIVE.value
```

Comment:
Inactive current PAD cfs_account

Reply:
Above is better than "# Make the current CFS Account as INACTIVE in DB"